jaeger-crossdock/build.gradle

```diff
@@ -13,9 +13,6 @@ compileJava {
 dependencies {
     compile project(':jaeger-client')
 
-    compile group: 'org.apache.thrift', name: 'libthrift', version: apacheThriftVersion
```

Comment:
+1, I think I had to add this in some iteration of the previous work, to get the tests to compile/run. Apparently, it's not needed now :)

Reply:
Er... oops :)

https://github.com/jaegertracing/jaeger-client-java/issues/494#issuecomment-405883005